src/core/a-entity.js

```diff
@@ -322,17 +361,24 @@ var proto = Object.create(ANode.prototype, {
    * @param {object} newData - The new properties assigned to the component
```

Comment:
I would mention what it means here if `elData === undefined` and what it means if `elData === null`